File: Id1/BruteForceSolutions/julia/brute.jl
function brute()::Int
    accumulator::Int = 0
    for i in 1:999
        if i % 3 == 0 || i % 5 == 0
            accumulator += i
        end
    end
    accumulator
end

function main()
    total_time::Float64 = 0.0
    num_iterations = 100
    result::Int = 0

    for _ in 1:num_iterations
        _, time_elap, _, _, _ = @timed result = brute()
        total_time += time_elap
    end

    average_time::Int = trunc(Int, (total_time/num_iterations)*1e9)

    println(result)
    println(average_time)
end

main()

File: Id1/BruteForceSolutions/java/Brute.java
public class Brute{
    public static long brute() {
        long accumulator = 0;
        for (int i = 0; i < 1000; i++) {
            if (i % 3 == 0 || i % 5 == 0) {
                accumulator += i;
            }
        }
        return accumulator;
    }

    public static void main(String[] args) {
        long result = brute();
        double total_time = 0;
        int num_iterations = 100;

        for (int i = 0; i < num_iterations; i++) {
            double startTime = System.nanoTime();
            brute();
            double endTime = System.nanoTime();
            double duration = endTime - startTime;
            total_time += duration;
        }

        long average_time = (long) (total_time / num_iterations);
        String averageTimeString = String.valueOf(average_time);
        
        System.out.println(result);
        System.out.println(averageTimeString);
    }
}